calc/src/mechanics/gen78.ts

```diff
@@ -825,17 +825,20 @@ export function calculateBPModsSMSS(
   );
 
   if (!move.isZ && !move.isMax && !noTypeChange) {
-    const normal = move.hasType('Normal');
-    if (attacker.hasAbility('Aerilate') && normal ||
-      attacker.hasAbility('Galvanize') && normal ||
-      attacker.hasAbility('Pixilate') && normal ||
-      attacker.hasAbility('Refrigerate') && normal ||
-      attacker.hasAbility('Normalize')
+    // The -ate abilities already changed move typing earlier
+    const normal = new Move(gen, move.originalName).hasType('Normal');
```

Comment:
Build error seems to be from this line. Guess I'll need to find a new way to check if the move was originally Normal-type.